PlmDrr/MRML/vtkMRMLPlmDrrNode.h

```diff
@@ -0,0 +1,166 @@
+/*==============================================================================
+
+  Program: 3D Slicer
+
+  Portions (c) Copyright Brigham and Women's Hospital (BWH) All Rights Reserved.
+
+  See COPYRIGHT.txt
+  or http://www.slicer.org/copyright/copyright.txt for details.
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+==============================================================================*/
+
+#ifndef __vtkMRMLPlmDrrNode_h
+#define __vtkMRMLPlmDrrNode_h
+
+// Beams includes
+#include "vtkSlicerPlmDrrModuleMRMLExport.h"
+
+// MRML includes
+#include <vtkMRML.h>
+#include <vtkMRMLNode.h>
+#include <vtkMRMLModelNode.h>
+
+// STD includes
+#include <list>
+
+class vtkMRMLLinearTransformNode;
+class vtkMRMLRTBeamNode;
+class vtkMRMLMarkupsClosedCurveNode;
+class vtkMRMLMarkupsFiducialNode;
+class vtkMRMLMarkupsLineNode;
+
+/// \ingroup SlicerRt_QtModules_PlmDrr
+class VTK_SLICER_PLMDRR_MODULE_MRML_EXPORT vtkMRMLPlmDrrNode : public vtkMRMLNode

```

Comment:
Can't you subclass or re-use the Planar Image node? It was created just for this purpose.

Reply:
I will do that, once everything will be settled.